manage.py

```diff
@@ -108,6 +108,12 @@ def seed():
     # Scrape articles for feed
     scrape_articles(Categorizer())
 
+    # Add items to user_item relationship
```

Comment:
Testing queries mostly. It doesn't cost much to fill up the table with one user and one set of feed items, so I see little reason not to do it.